### DISCUSSION TITLE
Symfony 6 add a database table row to exist table

### DISCUSSION THREAD

**Discussion by MindeMele** [symfony/symfony#47755] [Q&A]
How can I add to my products table a description row? Do I need to recreate this table?

**Comment by MindeMele** [comment]
I have this error:   An exception occurred while executing a query: SQLSTATE[HY000]: General error: 1 Cannot add a NOT NULL column with default value NULL

**Reply by Asmitta-01** [reply]
It means you are adding a row with an unauthorized emmty field.
How do you changed you Table and if there're some products inside you will have you delete all of them...
